file_manager/models.py

```diff
@@ -84,8 +86,8 @@ class Type(models.IntegerChoices):
         DELETE = 2
 
     type = models.IntegerField(choices=Type.choices)
-    position = models.IntegerField()
-    text = models.TextField()
+    position = models.IntegerField(blank=True, null=True)
```

Comment:
how it can be blank or null?

Reply:
if field is blank, then the field is null

Reply:
blank and null is different things.

Reply:
https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django